tests/publiccloud/xfsprepare.pm

```diff
@@ -69,8 +71,10 @@ sub create_config {
     assert_script_run("echo 'export SCRATCH_MNT=$mnt_scratch' >> $CONFIG_FILE");
     assert_script_run("echo 'export TEST_DEV=${device}1' >> $CONFIG_FILE");
     assert_script_run("echo 'export SCRATCH_DEV=${device}2' >> $CONFIG_FILE");
-    # Ensure reflink is enabled (required for several tests)
-    assert_script_run("echo 'MKFS_OPTIONS=\"-m reflink=1\"' >> $CONFIG_FILE");
+    # Add optional mkfs options
+    my $mkfs_options = get_var('XFS_MKFS_OPTIONS', '');
+    $mkfs_options .= "-m reflink=1" if (get_var('XFS_TESTS_REFLINK', 0) == 1);
```

Comment:
Maybe  with an extra space in case XFS_MKFS_OPTIONS cames without it?

`$mkfs_options .= " -m reflink=1"`

Reply:
Good catch! Updating it right away ....